framework/src/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -177,24 +178,34 @@ public StubParser(
         try {
             parsedStubUnit = JavaParser.parseStubUnit(inputStream);
         } catch (ParseProblemException e) {
-            ErrorReporter.errorAbort(
-                    "StubParser: exception from StubParser.parse for file "
+            StringBuilder message =
+                    new StringBuilder(
+                            "exception from StubParser.parse for file "
```

Comment:
After #1574 is merged, you could add the class currently being parsed.

Reply:
#1574 has been merged

Reply:
I merged in master and output the current class below.